rules/eslint/best-practices/on.js

```diff
@@ -1,43 +1,43 @@
 module.exports = {
     rules: {
         // Enforces getter/setter pairs in objects
-        'accessor-pairs': 0,
+        'accessor-pairs': 'off',
         // enforce return statements in callbacks of array’s methods
         // https://eslint.org/docs/rules/array-callback-return
-        'array-callback-return': [2, { allowImplicit: true }],
+        'array-callback-return': ['error', { allowImplicit: true }],
         // treat var statements as if they were block scoped
-        'block-scoped-var': 2,
+        'block-scoped-var': 'error',
         // specify the maximum cyclomatic complexity allowed in a program
         // https://eslint.org/docs/3.0.0/rules/complexity#limit-cyclomatic-complexity-complexity
-        complexity: [2, 11],
+        complexity: ['error', 11],
         // enforce that class methods use "this"
         // http://eslint.org/docs/rules/class-methods-use-this
-        'class-methods-use-this': 0,
+        'class-methods-use-this': 'off',
```

Comment:
What is the reasoning behind the filename here? `on`? It doesn't mean that the rules described in here should be on? Or is it just because you're doing them bit by bit?

Reply:
Allows for consumers of our config to turn entire rulesets "on" or "off". Yes, "on" may include rules that we don't actually want on, but it more just refers to the entire ruleset as opposed to specific rules themselves.

Basically just trying to conform to examples that I've seen ([like here](https://github.com/walmartlabs/eslint-config-walmart/tree/master/rules/eslint/best-practices)), which are great for users consuming your config.

Reply:
As in it's describing where we want to get to?

If the file name is `on` and the rule is `off` does that mean its finally off?
Likewise, file name `off` and rule is `on` is the rule on or off?

I think maybe I'm misunderstanding. 
![giphub](https://media1.giphy.com/media/3o7btPCcdNniyf0ArS/giphy.gif?cid=e1bb72ff5b7450f94f67496159107328)

Reply:
nah I think I'm just doing a bad job of explaining lol.

```
off.js => use this if you want to turn off all rules in a ruleset.
on.js => what rules in this ruleset 7Geese feels should be turned on, even if it's not all of them
```

Reply:
I think I'd like to hear the best words :)